### DISCUSSION TITLE
「 亲测 」更友好的 GitHub 体验

### DISCUSSION THREAD

**Discussion by uuutt2023** [share121/inter-knot#3110] [其他]
![ 封面 ](https://bbs-static.miyoushe.com/static/2023/11/16/e03be25a7b2bc86f2b1909dddb6ca915_6686505940047795068.png)

## 界面 - 中文化
![ 效果图 ](https://github.com/user-attachments/assets/7fc37f79-4db7-4420-bc19-6ca9565457f1)
[ 仓库 - github-chinese ](https://github.com/maboloshi/github-chinese)

## 镜像 - 直链下载
![ 效果图 ](https://github.com/user-attachments/assets/3a33ad7c-20f0-4ef7-935b-d11549297b64)
[ 脚本 - Github 增强 - 高速下载 ](https://greasyfork.org/zh-CN/scripts/412245-github-%E5%A2%9E%E5%BC%BA-%E9%AB%98%E9%80%9F%E4%B8%8B%E8%BD%BD)

**Comment by warning-Kirov-reporting** [comment]
蟹蟹你，但用不到<img src="https://bbs-static.miyoushe.com/static/2024/03/21/6431de7537491cb394a82e8a0ff8cf77_9185062232016213590.png" alt="丽都漫步-猫又" width="75" >

**Comment by Junjun280** [comment]
整挺好的，好用！
